package.json

```diff
@@ -29,9 +32,9 @@
     "wpcom-proxy-request": "^3.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
+    "start": "eslint src && react-scripts start",
```

Comment:
Yes, we could drop it, but I added it because CRA runs linting on start (and on file changes)